src/main/resources/log4j.properties

```diff
@@ -0,0 +1,30 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Just added for SBT run to see logging aside from the log hailstorm in spark-submit